src/tools/__tests__/decision_analysis.test.ts

```
@@ -1,161 +1,296 @@
+      decisionAnalysis({
+        options: ['Option'],
+        criteria: 'Not an array' as unknown as string[],
+        scores: [[5]],
+      })
+    ).rejects.toThrow(ValidationError);
+  });
+
+  it('throws ValidationError for empty string in options', async () => {
+    await expect(
+      decisionAnalysis(['Option A', ''], ['Criterion'], [[5], [5]])
+    ).rejects.toThrow(ValidationError);
```

ESLint

[error] Replace `⏎······decisionAnalysis(['Option·A',·''],·['Criterion'],·[[5],·[5]])⏎····).rejects.toThrow(ValidationError` with `decisionAnalysis(['Option·A',·''],·['Criterion'],·[[5],·[5]])).rejects.toThrow(⏎······ValidationError⏎····`

(prettier/prettier)